Provis.WebApi/Controllers/CalendarController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Provis.Core.DTO.CalendarDTO;
+using Provis.Core.Interfaces.Services;
+using Provis.Core.Roles;
+using Provis.WebApi.Policy;
+using System;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Provis.WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CalendarController : ControllerBase
+    {
+        protected readonly ICalendarService _calendarService;
+        private string UserId => User.FindFirst(ClaimTypes.NameIdentifier).Value;
+        public CalendarController(ICalendarService calendarService)
+        {
+            _calendarService = calendarService;
+        }
+
+        [Authorize]
+        [HttpPost]
+        [WorkspaceRoles(new WorkSpaceRoles[] {
+            WorkSpaceRoles.OwnerId, WorkSpaceRoles.ManagerId,
+            WorkSpaceRoles.MemberId})]
+        public async Task<IActionResult> CreateEventAsync([FromBody] EventCreateDTO eventCreateDTO)
+        {
+            await _calendarService.CreateEventAsync(eventCreateDTO, UserId);
+
+            return Ok();
+        }
+
+        [Authorize]
+        [HttpGet]
+        [WorkspaceRoles(new WorkSpaceRoles[] {
+            WorkSpaceRoles.OwnerId, WorkSpaceRoles.ManagerId,
+            WorkSpaceRoles.MemberId, WorkSpaceRoles.ViewerId})]
+        [Route("all/workspace/{workspaceId}")]
```

Comment:
It may be better to use "events" instead of "all"?

Reply:
![image](https://user-images.githubusercontent.com/93646070/153939766-dccacdf0-4dc5-4b0e-954b-e247c4b0980f.png)
I called by this way, because I need to divide two GET Requests, with same routes

Reply:
Maybe better to write "events" and "daily-events"?

Reply:
lets discuss it on daily meeting